src/tests/ftest/pool/RebuildTests.py

```diff
@@ -398,29 +313,21 @@ def test_multipool_rebuild(self):
             if pool2.pool_info.pi_rebuild_st.rs_errno != 0:
                 self.fail("Rebuild error reported: {}"
                           .format(pool2.pool_info.pi_rebuild_st.rs_errno))
-            if pool2.pool_info.pi_rebuild_st.rs_obj_nr != objcount:
+            if pool2.pool_info.pi_rebuild_st.rs_obj_nr != self.objcount:
                 self.fail("Rebuilt objs not as expected: {0} {1}"
-                          .format(pool2.pool_info.pi_rebuild_st.rs_obj_nr,
-                                  objcount))
-            if pool2.pool_info.pi_rebuild_st.rs_rec_nr != (reccount*objcount):
+                          .format(pool2.pool_info.pi_rebuild_st.rs_obj_nr, self.objcount))
+            if pool2.pool_info.pi_rebuild_st.rs_rec_nr != (self.reccount*self.objcount):
```

Comment:
(pylint-line-too-long) Line too long (88/80)

Reply:
no belong to current patch.